index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+    <title>JavaScript Mastery | web development course</title>
+  </head>
+  <body>
+    <header class="header-container">
+      <div class="circle"></div>
+      <nav>
+        <ul class="first-navbar">
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li><a class="firstbar" href="#">English</a></li>
+          <li><a class="firstbar" href="#">My account</a></li>
+          <li><a class="firstbar" href="#">logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <a href="index.html"
+          ><img class="brand-logo" src="img/logo.png" alt="Brand-name-and-logo"
+        /></a>
+        <ul class="nav-list-items">
+          <li class="list-item"><a href="index.html">Home</a></li>
+          <li class="list-item"><a href="Interfaces/about.html">About</a></li>
+          <li class="list-item"><a href="#">Program</a></li>
+          <li class="list-item"><a href="#">Sponsor</a></li>
+          <li class="list-item"><a href="#">New</a></li>
+          <li class="list-item"><a href="#">Ticket</a></li>
+        </ul>
+        <div class="hamburger-container">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+      </nav>
+    </header>
```

Comment:
- [ ] I think it would be better if you reduce the `height` of your header and the `spacing at the left of the logo` for both **Home** and **About** pages to stick as much as possible to the **[Behance template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)**.
### Template design
![Screenshot from 2022-09-22 12-05-40](https://user-images.githubusercontent.com/83042742/191720256-39e3c2b0-56db-474f-8c98-f690e4644181.png)

### Your design
![Screenshot from 2022-09-22 12-06-07](https://user-images.githubusercontent.com/83042742/191720183-f1e69fb1-5093-4d03-b8a2-9a5f8ed50733.png)